TestControllers/TestController.psm1

```diff
@@ -726,6 +726,10 @@ Class TestController {
 				$executionCount += 1
 				Write-LogInfo "($executionCount/$($this.TotalCaseNum)) testing started: $($currentTestCase.testName)"
 				Write-LogInfo "SetupConfig: { $(ConvertFrom-SetupConfig -SetupConfig $currentTestCase.SetupConfig) }"
+				# Retrieve VM info, when the VM doesn't have IP, we create new environemnt.
```

Comment:
This explanation is not enough to understand the problem. I don't understand why the VM has no IP at current point, and the logic is to `get` some information, not create new enviornment. Make the comments completed, and accurate.